flink-table-store-core/src/main/java/org/apache/flink/table/store/file/utils/FileUtils.java

```diff
@@ -182,4 +180,20 @@ public static Stream<Long> listVersionedFiles(Path dir, String prefix) throws IO
                 .filter(name -> name.startsWith(prefix))
                 .map(name -> Long.parseLong(name.substring(prefix.length())));
     }
+
+    public static BulkFormat.Reader<RowData> createFormatReader(
+            BulkFormat<RowData, FileSourceSplit> format, Path file) throws IOException {
+        if (!file.getFileSystem().exists(file)) {
+            throw new FileNotFoundException(
+                    String.format(
+                            "File '%s' not found, Possible causes: "
+                                    + "1.consumption is too slow, you can improve the performance of consumption. "
```

Comment:
For the users, "you can improve the performance of consumption." is hard to do something for improvement, which is better to tell users how to improve.